arbnode/batch_poster.go

```diff
@@ -1468,11 +1530,61 @@ func (b *BatchPoster) maybePostSequencerBatch(ctx context.Context) (bool, error)
 			l1BoundMinTimestampWithBypass = arbmath.SaturatingUSub(timestampWithPadding, arbmath.BigToUintSaturating(maxTimeVariationDelaySeconds))
 		}
 	}
+	var lastPotentialMsg *arbostypes.MessageWithMetadata
```

Comment:
can you put all this code in another function? Its to avoid merge conflicts later on when we merge future versions of nitro